controller/src/beerocks/master/db/node.h

```diff
@@ -292,8 +292,9 @@ class node {
     eTriStateBool client_stay_on_selected_band = eTriStateBool::NOT_CONFIGURED;
 
     // The selected bands the client should be steered to if the client_stay_on_selected_band is set.
-    // Default value is FREQ_UNKNOWN (also indicates value is not configured)
-    beerocks::eFreqType client_selected_bands = beerocks::eFreqType::FREQ_UNKNOWN;
+    // Default value is PARAMETER_NOT_CONFIGURED.
+    // Possible values are bitwise options of eClientSelectedBands.
+    int8_t client_selected_bands = beerocks::PARAMETER_NOT_CONFIGURED;
```

Comment:
to support "parameter not configured value"

I added a suggestion to out "fixups and improvements" PR to extend the eClientSelectedBands to have a "not configured" value - but since this change has a tree-wide effect - wanted to introduce it (id decided to) as hotfix commit (and of course change the db to hold eClientSelectedBands accordingly.